### ISSUE TITLE
Is there a way to display the scene hierarchy inside the viewer?

### ISSUE THREAD

**Issue by EXT-OWL** [UINOSOFT/t3d-model-viewer#3]
I'm pretty sure this is going to be straight-forward but I'm having a hard time implementing this - drop a model and along with the provided options add a box to display the scene hierarchy. Is this is a function that the t3d library has?

**Comment by EXT-OWL** [comment]
Nevermind - I found the code in the viewer that displays it. Its printGraph, correct?

**Comment by shawn0326** [comment]
Yes, I simply used `console.group` to output the tree structure on the console

**Comment by EXT-OWL** [comment]
thanks @shawn0326 .

How would I go about making a scene hierarchy which displays the nodes that I can toggle on and off?
